shells/lib/services/random.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import '../../../build/services/random.js';
```

Comment:
We don't need this file, or any of the redirects in shells/lib/services. 

I think this got started when Alex moved some stuff out of shells/lib/services and was kindly trying to change as little as possible. 

shells/lib/services exists only to contain actual services that require dependencies specific to shells.

Reply:
done